src/components/pill/pill-classic.style.js

```diff
@@ -46,8 +46,8 @@ export const classicStyleConfig = {
   }
 };
 
-export default (colourVariant, inFill, isDeletable) => {
-  const colourSet = classicStyleConfig[colourVariant];
+export default (colorVariant, inFill, isDeletable) => {
+  const colourSet = classicStyleConfig[colorVariant];
```

Comment:
Also just noticed colourSet here which should be updated too (sorry, totally missed this one before!)